src/styles/mixins.scss

```diff
@@ -0,0 +1,110 @@
+@mixin header-space-grotesk {
+  font-weight: 700;
+  font-style: normal;
```

Comment:
`font-style` is `normal` by default, so you don't need to set it manually everywhere.